index.html

```diff
@@ -0,0 +1,164 @@
+                <h3 class="details-date">2022.09.10 (SAT) ~ 13(MON)</h3>
+                <h3 class="details-text-two">@ Hampton Inn Manhattan-Times Square North</h3>
+            </div>
+        </section>
+
+        <!-- Main Program section -->
+        <section class="program-section">
+            <div class="main-program-header">
+                <h3>Main Program</h3>
+                <div class="main-line"></div>
+            </div>
+            <div class="program-proceedings">
+
+                <div class="program-Presentation">
+                    <img src="images/presentation.png" alt="presentations">
+                    <h4 class="program-titles">Lecture</h4>
+                    <p class="program-texts">Listen to all speakers from various regions and learn about trends
+                        in new normal business in all the world.</p>
+                </div>
+
+                <div class="program-Presentation">
+                    <img src="images/exibition.png" alt="presentations">
+                    <h4 class="program-titles">GS Exhibition</h4>
+                    <p class="program-texts">The world biggest PR awards programme, dedicated to benchmarking the best PR work.</p>
+                </div>
+
+                <div class="program-Presentation">
+                    <img src="images/forum.png" alt="presentations">
+                    <h4 class="program-titles">GS Forum</h4>
+                    <p class="program-texts">The biggest PR conference of the year, a high-level forum designed to address the critical issues that matter most.</p>
+                </div>
+
+                <div class="program-Presentation">
+                    <img src="images/workshop.png" alt="presentations">
+                    <h4 class="program-titles">GS workshop</h4>
+                    <p class="program-texts">A global network of conferences that explore the innovation and disruption that is redefining public relations.</p>
+                </div>
+
+                <div class="program-Presentation">
+                    <img src="images/ignite.png" alt="presentations">
+                    <h4 class="program-titles">GS Ignite</h4>
+                    <p class="program-texts">Unrivalled insight into the world's best PR agencies, across specialist and geographic categories to redefine.</p>
+                </div>
+
+            </div>
+            <div class="main-program-link"><a href="#">Join GS Summit 2022</a></div>
+        </section>
+
+        <!-- Featured Speakers Section -->
+        <section class="featured-speakers-section">
+            <div class="main-program-header">
+                <h3>Featured Speakers</h3>
+                <div class="main-line"></div>
+            </div>
+
+            <div class="speaker-proceedings">
+
+                <!-- DOM Section for featured speakers -->
+
+            </div>
```

Comment:
- [ ] In the speakers' section on the mobile screen, please fix the 'More' button alignment.
_Your design_
![Screenshot (2)](https://user-images.githubusercontent.com/47625407/176824469-5741310d-f899-47fd-ae5a-af42892c6bfd.png)